src/pages/Demo/demo.tsx

```diff
@@ -25,7 +25,6 @@ const Demo = ({ linkToDemo, density }: DemoPropsType) => {
         </>
       }
     >
-      <main className="overflow-hidden">
```

Comment:
Do you actually need to remove the main HTML element as well to achieve your expected goal of this PR? It's important to include general landmarks in the document.